robots/wikipedia.js

```diff
@@ -0,0 +1,152 @@
+const superAgent =  require('superagent');
+const readline = require('readline-sync');
+const unicode = require('unidecode');
+
+async function Wikipedia(content) {
+    const images = [];
+    var ctn = '';
+    var title = '';
+    var summary =''
+    var pageid = '';
+    var url = '';
+    const links = [];
+    const references = [];
+
+
+    console.log('Fetching from Wikipedia...')
+    var RealText = await getRealText(content.searchTerm)
+    title = RealText;
+    console.log('Searching content...')
+    await getContent();
+    console.log('Building Structure to others Robots...')
+    return await buildStructure();
+    
+    /*
+    *
+    * Tenta buscar o termo na Wikipedia, se o mesmo não for encontrado ele encerra o programa,
+    * Caso encontre mais de um,o mesmo sugere e pergunta qual termo você realmente tem interesse baseado na busca no Wikipedia.
+    * 
+    * Obs: Um fato interessante é que o mesmo caso mude a URL da wikipedia para https://pt.wiki... o mesmo vai retornar sua busca em Português Brasil
+    */
+    async function getRealText(text){
+        const res = await superAgent.get('https://en.wikipedia.org/w/api.php').query({
```

Comment:
Na minha humilde opinião, acredito que dessa forma sua busca trará resultados de páginas mais exatos.   
```suggestion
        const res = await superAgent.get('https://en.wikipedia.org/w/api.php').query({
            'action' : 'query',
            'list' : 'search',
            'srlimit' : 5,
            'srsearch' : '' + text,
            'format' : "json"
        })
```

Reply:
Ao invés de mais exatos, quis mesmo dizer mais variados.

Reply:
Então @genesluna acredito que teria que ser feito todo um tratamento, para estas sugestões seguindo esta sua sugestão, segue um exemplo:
![image](https://user-images.githubusercontent.com/19417162/54278203-82863d00-4570-11e9-9e00-9ea2d90599d9.png)

Nesta imagem é possivel visualizar que temos um problema que ao consultar a API por esta ação, ela retorna em seu titulo, algumas datas, sendo que seria nescessario fazer um tratamento para retirar as mesmas, da maneira que fiz hoje, não seria nescessario este tipo de tratamento.

Mas achei valida sua sugestão, seria possivel implementar desta maneira também, mas como disse antes fazendo este tratamento.

Valeu!! 🤙 🤙 🤙 🤙

Reply:
A ação que você está usando também retorna títulos com datas. Nesse caso as datas não são um problema, pois elas fazem parte do título da página hospedada na Wikipedia. E quando nós formos buscar o 'Extratc' da página, necessitaremos do título com as datas. O problema da ação que você está usando é que os resultados são limitados. A imagem a seguir mostra uma busca, com a ação que você usa, configurada para um limite de 100.

![Anotação 2019-03-13 094230](https://user-images.githubusercontent.com/1102418/54279965-1a862580-4575-11e9-95a0-c0876e71acac.png)

Como é possível observar, além da presença das datas, muitos resultados ficaram de fora. Por exemplo, se você quiser fazer um vídeo sobre a temporada 2017-2018 do Bayern (2017–18 FC Bayern Munich season) infelizmente não será possível, pois sua busca não retorna estes resultados. 

Como se pode inferir pela próxima imagem,  na ação que eu sugeri, você recebe estes dados, assim como muitos outros. Fica a critério do usuário escolher. 

![Anotação 2019-03-13 100904](https://user-images.githubusercontent.com/1102418/54281512-a483bd80-4578-11e9-8ba9-45992e4a6f73.png)

De qualquer forma, fico feliz que você tenha testado. 😃

Reply:
Justo @genesluna agora entendi a função destas datas!